parsec/api/data/certif.py

```diff
@@ -88,10 +88,10 @@ def is_admin(self) -> bool:
     @classmethod
     def verify_and_load(
         cls,
-        *args,
+        *args: Any,
```

Comment:
```suggestion
        *args: List[Any],
```
`*args` can be typed as a `list` right ?

Reply:
same as above

Reply:
ok